drivers/gpu/drm/tests/drm_format_helper_test.c

```diff
@@ -892,6 +892,96 @@ static void drm_test_fb_xrgb8888_to_mono(struct kunit *test)
 	KUNIT_EXPECT_MEMEQ(test, buf, result->expected, dst_size);
 }
 
+struct clip_offset_case {
+	const char *name;
+	unsigned int pitch;
+	u32 format;
+	struct drm_rect clip;
+	unsigned int expected_offset;
+};
+
+static struct clip_offset_case clip_offset_cases[] = {
+	{
+		.name = "pass through",
+		.pitch = TEST_USE_DEFAULT_PITCH,
+		.format = DRM_FORMAT_XRGB8888,
```

Comment:
It makes sense to use a format other than XRGB8888?

Reply:
My thought was that maybe someone would like to add tests for different formats. Because different formats could maybe affect the memcpy operation.

Reply:
Yeah, it makes sense. It was just out of curiosity.